src/components/NewSnippet.jsx

```diff
@@ -20,28 +20,35 @@ class NewSnippet extends React.Component {
       tags: [],
       syntax: '', // eslint-disable-line react/no-unused-state
     };
+    this.onKeyPress = (e) => {
+      if (e.which === 13) { e.preventDefault(); }
```

Comment:
Is there a constant for that? If not let's at least document, that `13` is a keycode of `Enter`